pkg/cmd/version/version.go

```diff
@@ -1,34 +1,58 @@
 package version
 
 import (
+	"fmt"
+	"io"
+	"os"
+
 	"github.com/jenkins-x/jx-helpers/v3/pkg/cobras/helper"
-	"github.com/jenkins-x/jx-helpers/v3/pkg/termcolor"
-	"github.com/jenkins-x/jx-logging/v3/pkg/log"
 	"github.com/spf13/cobra"
 )
 
 // Build information. Populated at build-time.
-var (
+type BuildInfo struct {
 	Version   string
 	Revision  string
 	Branch    string
 	BuildUser string
 	BuildDate string
 	GoVersion string
-)
+}
+
+//Instance of the build info
+var buildInfo BuildInfo
 
 const (
+	// VersionPrefix string for setting pre-releasedfdffddsfdsf etc
+	VersionPrefix = ""
+
+	// ExampleVersion shows an example version in the help
+	// if no version could be found (which should never really happen!)
+	ExampleVersion = "1.1.59"
 
 	// TestVersion used in test cases for the current version if no
 	// version can be found - such as if the version property is not properly
-	// included in the go test flags
-	TestVersion = "1.0.0-SNAPSHOT"
+	// included in the go test flags.
+	TestVersion = "3.2.238"
+
+	// TestRevision can be used in tests if no revision is passed in the test flags
+	TestRevision = "04b628f48"
+
+	// TestTreeState can be used in tests if no tree state is passed in the test flags
+	TestTreeState = "clean"
+
+	// TestBuildDate can be used in tests if no build date is passed in the test flags
+	TestBuildDate = "2020-05-31T14:51:38Z"
+
+	// TestGoVersion can be used in tests if no version is passed in the test flags
+	TestGoVersion = "1.13.8"
 )
 
 // ShowOptions the options for viewing running PRs
 type Options struct {
 	Verbose bool
 	Quiet   bool
```

Comment:
[You can do this in a separate PR if you want] Lets deprecate Quiet (dont remove it, as it is used downstream), and add a new short field which does the exact thing as quiet. Then set a date when it's scheduled for deletion. Also when an user tries --quiet or -q, let them know that the flag is deprecated and will be removed on a date, and they should instead use --short.

Quiet does not sound correct.

Reply:
The reason for this is only to make naming sounds better? Also, what is the data for deletion?

Reply:
> The reason for this is only to make naming sounds better? 

I have not seen many cli which have a quiet flag to only show version string. in helm/kubectl, it's always short (normally they have verbose to show more things). 

> Also, what is the data for deletion?

let's do 3 months - sort of a standard lifecycle to deprecate a cli flag imo.